api/app/signals/apps/signals/serializers.py

```diff
@@ -533,3 +540,28 @@ class Meta:
             'slug',
             'sub_categories',
         )
+
+
+#
+# Note objects field
+#
+
+# TODO: assess whether we should go the SelfLinkSerializerMixin route or the
+# NoteHyperlinkedRelatedField route
+class NoteHALSerializer(HALSerializer):
+    _signal = serializers.PrimaryKeyRelatedField(queryset=Signal.objects.all())
+    serializer_url_field = NoteHyperlinkedRelatedField
+
+    class Meta:
+        model = Note
+        fields = (
```

Comment:
I think it's handy to expose the `created_at` field as well? To show it in the FE.

Reply:
overgenomen